run/types/DeviceWrapper.ts

```diff
@@ -933,12 +1000,124 @@ export class DeviceWrapper implements SharedDeviceInterface {
     await this.setValueImmediate(text, element.ELEMENT);
   }
 
+  public async getAttribute(
+    attribute: string,
+    elementId: string
+  ): Promise<string> {
+    return this.toShared().getAttribute(attribute, elementId);
+  }
+
+  public async radioButtonSelected(timeOption: DMTimeOption) {
```

Comment:
maybe rename this to `disappearRadioButtonSelected` as it can only be used for that for now.

Reply:
done